.github/workflows/dev_builds.yml

```diff
@@ -460,6 +473,42 @@ jobs:
           path: ./install-script/install.ps1
           if-no-files-found: error
 
+  config-management-assets:
```

Comment:
To confirm, with this change these assets will now be under - [https://github.com/SumoLogic/sumologic-otel-collector/releases/latest/download/ansible-playbook](https://github.com/SumoLogic/sumologic-otel-collector/releases/latest/download/ansible-playbook.tar.gz)

Reply:
Correct.